packages/fxa-content-server/app/scripts/views/sms_send.js

```diff
@@ -24,6 +24,8 @@ import SmsMixin from './mixins/sms-mixin';
 import Template from 'templates/sms_send.mustache';
 import VerificationReasonMixin from 'views/mixins/verification-reason-mixin';
 
+const { FIREFOX_MOBILE_INSTALL } = SmsMessageIds;
```

Comment:
This messageId appears to be `undefined`, see https://screencap.co.uk/images/28bba7d14bbf21d9d84d5c40fa2909da5d9893bc.png

You can run this functional test locally using:

> npm run test-functional -- --grep="send_sms - valid phone number, resend"

Reply:
It's good @shane-tomlinson , this is undefined because of this https://github.com/mozilla/fxa/blob/3586aefe8cdee211220347895abe6e48b60a695f/packages/fxa-content-server/app/scripts/views/sms_sent.js#L14